x/gov/keeper/grpc_query_test.go

```diff
@@ -533,6 +710,96 @@ func (suite *KeeperTestSuite) TestGRPCQueryParams() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestLegacyGRPCQueryParams() {
+	queryClient := suite.legacyQueryClient
+
+	var (
+		req    *v1beta1.QueryParamsRequest
+		expRes *v1beta1.QueryParamsResponse
+	)
+
+	defaultTallyParams := v1beta1.TallyParams{
+		Quorum:        sdk.NewDec(0),
+		Threshold:     sdk.NewDec(0),
+		VetoThreshold: sdk.NewDec(0),
+	}
```

Comment:
I don't know why I need to do this - but the tests won't pass if I just use `v1beta1.TallyParams{}`